lib/mailparser.js

```diff
@@ -257,6 +261,49 @@ MailParser.prototype._process = function(finalPart) {
 
 };
 
+function force1MbLines(str) {
+    // this used to be: str.replace(/(.{1048576}(?!\r?\n|\r))/g, "$&\n");
+    // but regexp performance is terrible with such long strings
+    // the regexp uses "x(?!y)" which means "x not followed by y"
+
+    if (str.length < oneMb)
+        return str;
+
+    const oneMb = 1048576;
```

Comment:
it seems we use the `oneMb` variable a bit earlier https://github.com/frontapp/mailparser/pull/2/files#diff-fca333ac31758a01419c2620e3c6debcR269

Should we move it to the top to make things more clear ?